### PR TITLE
001 HR Team should be able to create a new sales employee now

### PR DESCRIPTION
**Pull request by ManpreetM01**

### CODE REVIEW

File: Create-Sales-Employee-Table-PB1.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+CREATE DATABASE Kainoos_ManpreetM;
+
+USE Kainoos_ManpreetM;
+
+-- ProductBacklog1
+CREATE TABLE SalesEmployees (
+	SalesEmployeeID int,
```
Comment (TomBillingtonUK) [inline Create-Sales-Employee-Table-PB1.sql]: Is there anything else we may want to do with the sales employee id? Think about protecting the integrity of this value across multiple data rows

File: Create-Sales-Employee-Table-PB1.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    Name varchar(70),
+    Salary decimal(10,2), 
+    BankAccountNumber varchar(8),
+    NationalInsuranceNumber char(9),
+    CommissionRate float);
```
Comment (TomBillingtonUK) [inline Create-Sales-Employee-Table-PB1.sql]: Is a float precise enough for this column as this data will be used to calculate/manipulate a monetary value

**Review by TomBillingtonUK — COMMENTED** [review]
Here are a couple of things to think about. I would also add the backlog item number to the PR title to help keep it clear

File: Create-Sales-Employee-Table-PB1.sql (hunk 1 above)
Comment (TomBillingtonUK) [reply Create-Sales-Employee-Table-PB1.sql]: You may want to think about protecting the integrity of the other columns as well

**Review by TomBillingtonUK — APPROVED** [review]

**Review by sammy-perry-kainos — APPROVED** [review]

**Pull request merged**